var/spack/repos/builtin/packages/vacuumms/package.py

```diff
@@ -16,25 +16,48 @@ class Vacuumms(CMakePackage):
     under Linux on x86_64 and ARM64. Please submit questions, pull requests, and bug
     reports via github. https://dl.acm.org/doi/abs/10.1145/2335755.2335826"""
 
-    homepage = "https://github.com/frankwillmore/VACUUMMS"
-    url = "https://github.com/frankwillmore/VACUUMMS/archive/refs/tags/v1.0.0.tar.gz"
-    git = "https://github.com/frankwillmore/VACUUMMS.git"
+    homepage = "https://github.com/VACUUMMS/VACUUMMS"
+    url = "https://github.com/VACUUMMS/VACUUMMS/archive/refs/tags/v1.0.0.tar.gz"
+    git = "https://github.com/VACUUMMS/VACUUMMS.git"
 
     maintainers("frankwillmore")
 
-    license("Unlicense")
+    license("MIT")
+
+    # This is the main branch, for the latest functionality
+    version("develop", branch="develop")
 
-    version("master", branch="master")
-    version("1.1.1", tag="v1.1.1", commit="a395930c87f5b52ab0ab8db278bd2bde592e8f12")
     version(
         "1.0.0",
         sha256="c18fe52f5041880da7f50d3808d37afb3e9c936a56f80f67838d045bf7af372f",
         deprecated=True,
     )
+    version(
+        "1.1.4",
+        sha256="13052bafca8622df5f14613b3d4cddd50bae725e0c13c1df7f3cdee28a2e673a",
+        deprecated=False,
+    )
+    version(
+        "1.2.0",
+        sha256="1836e6c29d1e83a79f9c6a9b686d1419bd72618aba7c46d3d038373808519d56",
+        preferred=True,
+        deprecated=False,
+    )
```

Comment:
The packaging convention is to order version directives from newest to oldest so please reverse the ordering of these.  (See the first note in the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching.)